src/org/jetbrains/plugins/scala/project/template/Downloader.scala

```diff
@@ -61,6 +62,14 @@ object Downloader {
     case Platform.Dotty => Seq(
       s"""set libraryDependencies := Seq("ch.epfl.lamp" % "dotty_2.11" % "$version" % "scala-tool")""",
       "updateClassifiers")
+
+    case Platform.Hydra => Seq(
+      s"""set scalaVersion := "${version.split("_")(0)}"""",
+      s"""set credentials := Seq(Credentials("Artifactory Realm", "repo.triplequote.com", "${HydraCredentialsManager.getLogin}", "${HydraCredentialsManager.getPlainPassword}"))""",
+      s"""set resolvers := Seq(Resolver.url("Triplequote Plugins Ivy Releases", url("https://repo.triplequote.com/artifactory/ivy-releases/"))(Resolver.ivyStylePatterns), Resolver.url("Triplequote sbt-plugin-relseases", url("https://repo.triplequote.com/artifactory/sbt-plugins-release/"))(Resolver.ivyStylePatterns),  "Triplequote Plugins Releases" at "https://repo.triplequote.com/artifactory/libs-release-local/")""",
```

Comment:
the repo URL is duplicated, so we need to fix that (and make it configurable)

Reply:
@dragos Unfortunately, I don't think there is any duplicate URL here. The only thing to correct is to use `libs-release` instead of `libs-release-local`.

* ivy-releases is to download hydra
* sbt-plugins-release is to download hydra-bridge
* libs-release is to download scala-hydra

Reply:
@dragos Maybe it would be better to create an issue for this.

Reply:
I guess I was referring to the hostname (`repo.triplequote.com`). But I understand it's a bit complicated. If we publish everything to Maven, things would become a lot simpler, right?